package.json

```diff
@@ -21,13 +21,14 @@
   },
```

Comment:
I've seen that all tests are run everytime a commit is made to the main repo. Cool!
Is this a good practice or is there a motivation for it?

Reply:
Runing unit and integration tests for every commit is a good practice because this way we can ensure that every commit has no fails before runing the tests on travis (there will run e2e, unit and integration tests). In the future, if unit and integration tests are too slow, we will have to think about it

Reply:
That is fine for me.
Thanks!

Reply:
I've followed the instructions for the 'html5/froala3' testing and it all worked fine. :smiley: :clinking_glasses: 

On the other hand, I've detected that the 'npm serve' command doesn't works on the 'html5/froala3' demo.

The 'npm start' works, though. :thinking: 
Maybe, it's only happening on my local machine.

This is the Terminal log.

``` sh
➜  froala3 git:(KB-3042) ✗ npm serve 

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    /home/dcanet/.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.14.7 /home/dcanet/npm-global/lib/node_modules/npm
```